.circleci/config/jobs/test-ui.yml

```diff
@@ -24,4 +24,8 @@ steps:
       command: cd ui && yarn run lint:hbs
   - run:
       name: Ember tests
-      command: cd ui && yarn test
+      command: mkdir -p /tmp/test-reports && cd ui && yarn test
```

Comment:
This directory could also be generated in `testem.js`, I’m ambivalent

Reply:
I'm also ambivalent.